src/main/java/graphql/execution/Async.java

```diff
@@ -23,15 +24,47 @@
 @SuppressWarnings("FutureReturnValueIgnored")
 public class Async {
```

Comment:
This is actually the code from https://github.com/graphql-java/graphql-java/pull/3462

Which is approved